x/multisig/types/keygen.go

```diff
@@ -67,40 +77,41 @@ func (m KeygenSession) GetKeyID() exported.KeyID {
 }
 
 // AddKey adds a new public key for the given participant into the keygen session
-func (m *KeygenSession) AddKey(blockHeight int64, participant sdk.ValAddress, pubKey exported.PublicKey) error {
+func (m *KeygenSession) AddKey(blockHeight int64, participant sdk.ValAddress, pubKey exported.PublicKey) (bool, error) {
```

Comment:
this return doesn't make much sense to me. Would you understand why `AddKey` returns a bool and an error if you saw this signature? Can't we get this another way?

Reply:
one alternative is what i mentioned in the above comment. Open to other suggestions